images/base/files/usr/local/bin/entrypoint

```diff
@@ -150,8 +150,10 @@ select_iptables() {
   fi
 
   echo "INFO: setting iptables to detected mode: ${mode}" >&2
+  if [ -x /usr/bin/update-alternatives ]; then
```

Comment:
not setting the correct mode is an issue though, in BOTH directions

Reply:
Turns out that this functionality is handled by a simple symlink